src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -299,8 +316,13 @@ protected Attribute GetDefaultAttribute(Type attributeType)
         /// Determines if a specified attribute is the same as an attribute
         /// in the collection.
         /// </summary>
-        public bool Matches(Attribute attribute)
+        public bool Matches(Attribute? attribute)
         {
+            if (attribute == null)
```

Comment:
I decided to be consistent with `Contains`. Let me know what you think

Reply:
I don't think we should change this.  To my knowledge we've received 0 complaints about null not being allowed.  I'd just revert your changes to this method.